docker-compose.yml

```diff
@@ -46,6 +48,10 @@ services:
     << : *default-volumes
     environment:
       << : *default-environment
+    # Mount volumes from Pygmy to inject host SSH key into container.
+    # https://pygmy.readthedocs.io/en/master/ssh_agent/
+    volumes_from: ### Local overrides to mount host SSH keys. Automatically removed in CI.
```

Comment:
@steveworley could you please confirm that "Automatically removed in CI." is still the case. If not - we need to remove this as it is very confusing (to me, this means that CI does some adjustments to the build process in some other "secret" places).